test/parser/test_parser_utils.py

```diff
@@ -211,6 +211,15 @@ def test_in_subquery(self):
                               {'table_name': 'treatment', 'alias': '',
                                'filter_column_list': [{'column_name': 'treatmentname', 'opt': '='}]}]
 
+    def test_update_parameterize(self):
+        sql = """
+            UPDATE `t1` SET `c`='11' WHERE (`id`='1111111')          
+        """
+        statement_node = ParserUtils.parameterized_query(parser.parse(sql))
+        statement = format_sql(statement_node, 0)
+        assert statement == """UPDATE t1 SET c = ? WHERE id = ?
+"""
```

Comment:
Will the newline characters change before and after sql parameterization?

Reply:
It is expected at present, for the front-end display, but it may be best to remove the last line. I will open another issue to solve this problem